fullstop-violation/fullstop-violation-persister-jpa/src/main/resources/db/migration/V16__create_lambda_function_created_from_untrusted_location_type.sql

```diff
@@ -0,0 +1,20 @@
+INSERT INTO fullstop_data.violation_type (
+  id,
+  help_text,
+  violation_severity,
+  title,
+  last_modified_by,
+  last_modified,
+  created,
+  created_by,
+  version)
+VALUES ('LAMBDA_FUNCTION_CREATED_FROM_UNTRUSTED_LOCATION',
+        'The Lambda function you created/updated is not stored in our trusted S3 bucket.',
```

Comment:
'The code that has been used in a Lambda function, is not stored in of the trusted S3 buckets.'

Reply:
I would actually like to also put a link to some documentation there. But the docs will only be internal :-/